README.md

```diff
@@ -51,7 +49,7 @@ as well as the **SOPT** installation:
         conan create ./cppflow/
         ```
       Note that conan requires you to specify the host (h) and the build (b) profiles on the command
-      line (`-pr:h=default -pr:b=default`), unless you have defined them in your conan profile.
+      line (`-pr=default`), unless you have defined them in your conan profile.
```

Comment:
I'm not sure why this changed; do we not need to set both the build and the host profiles?

Reply:
I thought `-pr` takes care of both, so the wording might just need a tweak 🤔

Reply:
I think that `-pr` only does the host:
`  -pr PROFILE_HOST, --profile PROFILE_HOST
                        Apply the specified profile to the host machine
  -pr:b PROFILE_BUILD, --profile:build PROFILE_BUILD
                        Apply the specified profile to the build machine
  -pr:h PROFILE_HOST, --profile:host PROFILE_HOST
                        Apply the specified profile to the host machine`

(from `conan create -h`)

Reply:
Ok, clarified now.